client/src/actions/moviesFetchRequested.test.js

```diff
@@ -0,0 +1,19 @@
+import {MOVIES_FETCH_REQUESTED} from './actionTypes';
+import {moviesFetchRequested} from './moviesFetchRequested';
+import SEARCH_FILTER_TYPE from '../components/SearchFilter/searchFilterType';
+import SORT_TYPE from '../components/Sort/sortType';
+
+it('should return action MOVIES_FETCH_REQUESTED', () => {
+    const filterType = SEARCH_FILTER_TYPE.genre;
```

Comment:
_trivial_

There is no totally wrong here, but I'd like you to to take a step back and look around, wouldn't it be much more readable to be more explicit about values in the test case.

```javascript
  const filterType = 'testFilterType';
  const filterValue = 'Comedy';
  const sortType = 'releaseDate';

  const expectedAction = {
    type: 'MOVIES_FETCH_REQUESTED',
    payload: {
      filterType: 'testFilterType,
      filterValue: 'Comedy',
      sortType: 'releaseDate',
    }
  };

  const moviewFetchRequestedAction = moviesFetchRequested(filterType, filterValue, sortType);

  expect(moviewFetchRequestedAction).toEqual(expectedAction);
```